tests/cli/test_flow_run.py

```diff
@@ -16,6 +28,51 @@ async def assert_flow_run_is_deleted(orion_client, flow_run_id: UUID):
         await orion_client.read_flow_run(flow_run_id)
 
 
+def assert_flow_runs_in_result(result, expected, unexpected=None):
+    output = result.stdout.strip()
+
+    # When running in tests the output of the columns in the table are
+    # truncated, so this only asserts that the first 20 characters of each
+    # flow run's id is in the output.
```

Comment:
We shouldn't ever be truncating ids in CLI output, right? Can we fix that? Could be a separate issue.

Reply:
Hm. We disable soft-wrap during test runs https://rich.readthedocs.io/en/stable/console.html#soft-wrapping which should prevent cropping. I'm fine with adding an issue to the backlog.

Reply:
https://github.com/PrefectHQ/prefect/issues/6311